drivers/video/ov2640.c

```diff
@@ -463,16 +459,14 @@ struct ov2640_data {
 	struct video_format fmt;
 };
 
-#define OV2640_VIDEO_FORMAT_CAP(width, height, format) \
-	{ \
-		.pixelformat = (format), \
-		.width_min = (width), \
-		.width_max = (width), \
-		.height_min = (height), \
-		.height_max = (height), \
-		.width_step = 0, \
-		.height_step = 0 \
-	}
+#define OV2640_VIDEO_FORMAT_CAP(width, height, format)                                             \
+	{.pixelformat = (format),                                                                  \
+	 .width_min = (width),                                                                     \
+	 .width_max = (width),                                                                     \
+	 .height_min = (height),                                                                   \
+	 .height_max = (height),                                                                   \
+	 .width_step = 0,                                                                          \
+	 .height_step = 0}
```

Comment:
Here, `clang-format` did not wrap the braces in the conventional style.
This can be controlled by adding a trailing `,` after the `0` on the last line, before the `}`, and re-applying clang-format again.